quay-enterprise/tectonic/files/quay-servicetoken-role-k8s1-6.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:  
+  name: quay-enterprise-serviceaccount
+  namespace: quay-enterprise
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - put
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
```

Comment:
Hi Eric,
I've taken the RBAC roles for 1.5 and changes the `apiVersion` from rbac alpha to beta. From Quay's perspective, it needs to have access to modify a secret object that gets created in the quay-enterprise namespace which QE stores it's configuration at.